src/app/attendance/actions/CreateAttendance.tsx

```diff
@@ -0,0 +1,21 @@
+"use server";
```

Comment:
file name needs to be lowercase. Let's call it `create-attendance.ts`. Also, it's not tsx, its ts.

Reply:
Fixed